.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node
+sudo: required
+dist: trusty
+language: node_js
+node_js:
+  - "7"
+matrix:
+  fast_finish: true
```

Comment:
I guess you don't need this line. This only is required when you have allowed_failure matrix